staff/mykhaylo-gusak/skylab-semantics/README.md

```diff
@@ -0,0 +1,506 @@
+# Skylab Semantics
```

Comment:
@MichaelXXI this file should not be in here! this is not from this branch. Please remove it from this PR asap and inform us when it's done. Thanks.